lib/matplotlib/lines.py

```diff
@@ -1464,7 +1464,7 @@ def is_dashed(self):
         return self._linestyle in ('--', '-.', ':')
 
 
-class _AxLine(Line2D):
+class AxLine(Line2D):
```

Comment:
It looks like this needs to be added to `doc/api/lines_api.rst` to be public and linkable.